### PR TITLE
Completed prep-javascript-arrays.

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
JS_Array

### CODE REVIEW

File: prep-javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const thirdElement = languages[2];
+
+console.log(thirdElement);
+
+const length = languages.length;
+
+console.log(length);
+
+const lastElement = languages[5];
```
Comment (BrettAlbright) [inline prep-javascript-arrays/main.js]: using the value works, but we want a dynamic solution. You want to use the length variable here as noted in the instructions and subtract one from it to target the final index

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
nice work, just one update noted

File: prep-javascript-arrays/main.js (hunk 1 above)
Comment (j-alejandro-araujo) [reply prep-javascript-arrays/main.js]: Adjusted, thank you! 

adjustment

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**